src/cryptojwt/key_bundle.py

```diff
@@ -751,7 +751,7 @@ def difference(self, bundle):
 
         return [k for k in self._keys if k not in bundle]
 
-    def dump(self):
+    def dump(self, cutoff: Optional[list] = None):
```

Comment:
Somewhat confusing attribute name (at least for me). Perhaps `excluded_attrs` is better?

The type should be `Optional[List[str]]` I think (I assume attribute names are strings)

Reply:
Horticultural term :-)
Imaging that you are dealing with something that looks like a tree and there are branches you don't want to be included so you cut them off. 
Yes, attribute names are strings.